postgresql-11.yaml

```diff
@@ -107,10 +108,27 @@ subpackages:
         - bash
         - pgaudit-11
     pipeline:
-      - uses: bitnami/compat
-        with:
-          image: postgresql
-          version-path: 11/debian-11
+      - runs: |
+          # right before the postgresql/11/debian-11 directory was deleted
+          # https://github.com/bitnami/containers/commits/main?after=69569bab0f3fc09fe53829e620228adef40275cf+209&branch=main&qualified_name=refs%2Fheads%2Fmain
+          # Deprecate PostgreSQL 11 and Odoo 14
+          git clone https://github.com/bitnami/containers.git bitnami
+          cd bitnami
+          git checkout 41ba7ded693dcaa167a47761230c50e01e71f9eb
```

Comment:
Would it make sense to add a `commit` input to the bitnami/compat pipeline? https://github.com/wolfi-dev/os/blob/e144eecd5fb13f43e09c42c7ab47a9f8bf222716/pipelines/bitnami/compat.yaml#L7

Reply:
Yeah that'd be great!

Reply:
I couldn't use the pipeline with commit because git-checkout pipeline truncates the history so I'm having an error like:

```
 fatal: reference is not a tree <commit>
```

Reply:
Yeah I guess you'd need to override the depth to include that commit, which is a little awkward.